docs/tutorials/ultradns.md

```diff
@@ -191,9 +191,9 @@ Once the service has an external IP assigned, ExternalDNS will notice the new se
 
 ## Verifying UltraDNS Records
 
-Please verify on the [UltraDNS UI](https://portal.ultradns.neustar) that the records are created under the zone "example.com".
+Please verify on the [UltraDNS UI](https://portal.ultradns.com/login) that the records are created under the zone "example.com".
 
-For more information on UltraDNS UI, refer to (https://docs.ultradns.neustar/mspuserguide.html).
+For more information on UltraDNS UI, refer to (https://docs.ultradns.com/menu.html).
```

Comment:
This one doesn't appear to go anywhere useful. Perhaps it requires being logged in?

Reply:
@kundan2707 Can you fix this link ? The other looks ok, I think we can merge this PR once you fixed this one.

Reply:
@mloiseleur updated link to  redirect correct page.